packages/security-cordova/plugin.xml

```diff
@@ -8,5 +8,6 @@
     <name>cordova-plugin-aerogear-security</name>
 
     <dependency id="cordova-plugin-iroot" version="0.7.0" />
+    <dependency id="cordova-plugin-device" version="2.0.2">
```

Comment:
I will need to test it as we already using this version of the plugin in the metrics-cordova. Not sure if having both will not clash or cause some issues.